www/pages/stop/stop.html

```diff
@@ -3,7 +3,7 @@
   <ion-nav-buttons side="secondary">
     <button id="like" data-tap-disabled="true" ng-class="{'button button-icon icon ion-ios-heart': liked === true, 'button button-icon icon ion-ios-heart-outline': liked === false}"
       ng-click="liked= !liked; toggleHeart(liked)"
-      aria-label="Like">
+      aria-label="{{liked ? 'Remove from liked' : 'Add to liked'}}">
```

Comment:
Should make the phrasing here consistent with that in the RouteDirective's and StopDirective's heart icons.